spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/BinderHeaders.java

```diff
@@ -40,6 +40,12 @@
 	public static final String PARTITION_HEADER = "partition";
 
 	/**
+	 * Indicates the target partition of an outbound message. Overrides
+	 * any partition selected by the binder.
+	 */
+	public static final String PARTITION_OVERRIDE = "partitionOverride";
```

Comment:
Not related to the subject, but why there is no `prefix`?
Imagine downstream application which has some logic around headers.
So, how to distinguish binder headers from all others?
Or just how to understand that the header in target message is related to binder not some other "hell"?

Thanks